src/components/html/EcAppCard.html

```diff
@@ -1,25 +1,11 @@
-<a-card class="ec-app-card" @click="$emit('click')" hoverable>
-  <div slot="cover" class="text-center">
-    <img
-      v-if="iconUrl"
-      class="img-fluid"
-      :src="iconUrl"
-      :alt="app.title"
-    />
-    <a-icon v-else type="picture"/>
+<div class="card" @click="$emit('click')" hoverable style="height: 26rem">
+  <img v-if="iconUrl" class="card-img-top" :src="iconUrl" :alt="app.title" />
```

Comment:
Aproveitamos o refactor pra melhorar code style:
https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended
> Elements with multiple attributes should span multiple lines, with one attribute per line.

```suggestion
<div
  class="card"
  @click="$emit('click')"
  style="height: 26rem"
  hoverable
>
  <img
    v-if="iconUrl"
    class="card-img-top"
    :src="iconUrl"
    :alt="app.title"
  />
```

Reply:
Aqui ainda, o `hoverable` era uma prop do `<a-card>`, não é um attr válido de `<div>`, então deveria ser retirado também

Reply:
E mais um ponto, o `<a-icon v-else type="picture"/>` você engoliu com angu aqui :grin:
Aí já é uma alteração na funcionalidade mesmo então é mais importante, isto é um "placeholder" pra apps sem imagem, tente substituir por algo do tipo com Bootstrap e FontAwesome 5